spec/index.bs

```diff
@@ -2210,20 +2180,125 @@ enum GPUTextureAspect {
+
+                                    : {{GPUTextureViewDimension/"cube"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `6`.
+                                    :: |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/width=] is
+                                        |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/height=].
+
+                                    : {{GPUTextureViewDimension/"cube-array"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"2d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is a multiple of `6`.
+                                    :: |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/width=] is
+                                        |this|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/height=].
+
+                                    : {{GPUTextureViewDimension/"3d"}}
+                                    :: |this|.{{GPUTexture/[[dimension]]}} is {{GPUTextureDimension/"3d"}}.
+                                    :: |descriptor|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `1`.
+                                </dl>
+                        </div>
+
+                        Then:
+                            1. Generate a {{GPUValidationError}} in the current scope with appropriate error message.
+                            1. Create a new [=invalid=] {{GPUTextureView}} and return the result.
+
+                    1. Let |view| be a new {{GPUTextureView}} object.
+                    1. Set |view|.{{GPUTextureView/[[texture]]}} to |this|.
+                    1. Set |view|.{{GPUTextureView/[[descriptor]]}} to |descriptor|.
+                    1. Return |view|.
+                </div>
+        </div>
 </dl>
 
+Issue: Define what it means to be a <dfn dfn>compatible texture format</dfn> with another texture format.
+
+<div algorithm>
+    When <dfn abstract-op>resolving GPUTextureViewDescriptor defaults</dfn> for {{GPUTextureViewDescriptor}}
+    |descriptor| run the following steps:
+
+    1. Let |resolved| be a copy of |descriptor|.
+    1. If |resolved|.{{GPUTextureViewDescriptor/format}} is `undefined`,
+        set |resolved|.{{GPUTextureViewDescriptor/format}} to |texture|.{{GPUTexture/[[format]]}}.
+    1. If |resolved|.{{GPUTextureViewDescriptor/mipLevelCount}} is `undefined`,
+        set |resolved|.{{GPUTextureViewDescriptor/mipLevelCount}} to |texture|.{{GPUTexture/[[mipLevelCount]]}}
+        &minus; {{GPUTextureViewDescriptor/baseMipLevel}}.
+    1. If |resolved|.{{GPUTextureViewDescriptor/arrayLayerCount}} is `undefined`,
+        set |resolved|.{{GPUTextureViewDescriptor/arrayLayerCount}} to
+        |texture|.{{GPUTexture/[[textureSize]]}}.[=Extent3D/depth=] &minus;
+        {{GPUTextureViewDescriptor/baseArrayLayer}}.
```

Comment:
This seems like it should only apply to 2d textures (probably regardless of the dimension of the view - 2d/2d-array/cube/cube-array). Though I see this was an existing issue.